### ISSUE TITLE
Mod compatibility

### ISSUE THREAD

**Issue by SylisMC** [Andrew6rant/InventoryTabs#6]
Would you consider adding mod support for other popular mods, or give us the ability to manually add blocks to a list for being detected by the inventory tabs via a config file?

**Comment by Andrew6rant** [comment]
Sorry I missed this message, I forgot that forks don't have notifications enabled by default. A config system is something I'd like to do in the future. There are ways mod authors can currently add support for InventoryTabs (see the README), but it is a bit of a convoluted process.

**Comment by KavisKryptic** [comment]
Is there a way for a modpack creator to do so? I read the config but unfortunately I'm still really amateur in Java and reading/understanding the JSON formatting. Although if you could point me in the right direction, (basically explain some things in layman's terms, or point me to some resources that would make it so I understand the README) I'd *really* appreciate it. 

Since I have tons of mods in my modpack that have their own chests, furnaces, and so on, and would love to make tabs for them.

**Comment by Andrew6rant** [comment]
Unfortunately, not really. Tab support has to be added manually. It's something that I am going to attempt to change in the future
You can send me a list of mods with inventories and I'll see what I can do to support them

**Comment by Andrew6rant** [comment]
Nevermind, I figured out a way to dynamically generate tabs for mod support!

Unless the mod does something funky with screens (like ExpandedStorage), InventoryTabs should support most mods out of the box. I'll attempt to manually add support for ones that don't work.

@KavisKryptic, try your modpack again with version 0.6.0 of InventoryTabs and see if more mods are supported

**Comment by KavisKryptic** [comment]
@Andrew6rant Just tried it right now with multiple chest-like/storage blocks, works like a charm! Thank you so much!

The only one that isn't working as it's supposed to is Iron Chests: Restocked. And that's because it's one of those that does " something funky with screens" as you put it lol.

**Comment by Andrew6rant** [comment]
Version 0.7.0 should add support for all inventories, even ones that do custom rendering (Expanded Storage, Iron Chests: Restocked, TechReborn, etc.)
